minimal/app/root.tsx

```diff
@@ -0,0 +1,25 @@
+import { Links, Meta, Outlet, Scripts, ScrollRestoration } from "react-router";
+
+import type { Route } from "./+types/root";
+
+export function Layout({ children }: { children: React.ReactNode }) {
+  return (
+    <html lang="en">
+      <head>
+        <meta charSet="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <Meta />
+        <Links />
+      </head>
+      <body>
+        {children}
+        <ScrollRestoration />
+        <Scripts />
+      </body>
+    </html>
+  );
+}
+
+export default function App({}: Route.ComponentProps) {
+  return <Outlet />;
+}
```

Comment:
The default template has a default `ErrorBoundary` - maybe we oughta keep that in this minimal template?

Reply:
Added